controllers/v1/bulkUploadRequest.js

```diff
@@ -5,159 +5,369 @@
  * Description : bulk upload operations 
  */
 
-const bulkUploadHelper = require(MODULES_BASE_PATH + "/bulk-upload-request/helper.js");
- 
+const bulkUploadHelper = require(MODULES_BASE_PATH + "/bulkUploadRequest/helper.js");
+
+/**
+   * BulkUploadRequest
+   * @class
+*/
+module.exports = class BulkUploadRequest extends Abstract {
+  constructor() {
+    super(schemas["bulkUploadRequest"]);
```

Comment:
@rakeshSgr - Let's use the plural, i.e. BulkUploadRequests, same for model and mongo collection

Reply:
resolved